dcpy/utils/postgres.py

```diff
@@ -308,19 +308,28 @@ def insert_dataframe(
                         table_name=AsIs(table_name),
                         conn=conn,
                     )
-                df.to_sql(
-                    table_name,
-                    schema=schema or self.schema,
-                    con=conn,
-                    if_exists=if_exists,
-                    index=False,
-                    dtype={
-                        "geo_1b": dialects.postgresql.JSON,
-                        "geo_bl": dialects.postgresql.JSON,
-                        "geo_bn": dialects.postgresql.JSON,
-                    },
-                    method=insert_copy,
-                )
+                if isinstance(df, gpd.GeoDataFrame):
+                    df.to_postgis(
```

Comment:
love it, getting a GeoDataFrame into postgis has always been a pain with `df.to_sql`

Reply:
Getting it back out is even more of a pain as it turns out

Reply:
(without manually specifying a geometry column name)